src/components/List/Cells/OutcomeCell.tsx

```diff
@@ -1,7 +1,27 @@
 import React from 'react'
-import { Segment } from 'semantic-ui-react'
 import { CellProps } from '../../../utils/types'
+import { Label, SemanticCOLORS, SemanticICONS } from 'semantic-ui-react'
+import strings from '../../../utils/constants'
+import { ApplicationOutcome } from '../../../utils/generated/graphql'
 
-const OutcomeCell: React.FC<CellProps> = ({ application }) => <p>{application.outcome}</p>
+const outcomeDisplayMap: {
+  [key in ApplicationOutcome]: { icon: SemanticICONS; color: SemanticCOLORS; text: string }
+} = {
+  PENDING: { icon: 'hourglass half', color: 'grey', text: '' }, // Not used
```

Comment:
Shouldn't we show something for pending?

Reply:
I think from a user's point of view, having no outcome makes sense -- outcome implies "finished". We're kind of just using "Pending" as an internal state for our purposes, but don't think we need to show it to the user.

Also, I figured that the majority of the time, people are going to be looking at active applications, so no need to have a million badges everywhere. I just wanted the completed applications to stand out from the rest.

I'm open to better ways to style this, but I think it requires a bit more design thought. (one idea was to shade the row background to show different outcome states, but I didn't want to go too far down this road without a designer having a crack)